libs/chain-signatures/contract/tests/common.rs

```diff
@@ -674,3 +685,34 @@ pub async fn submit_participant_info(
         .await?;
     Ok(result.is_success())
 }
+
+pub async fn assert_running_return_participants(
+    contract: &Contract,
+) -> anyhow::Result<Participants> {
+    // Verify contract is back to running state with new threshold
+    let final_state: ProtocolContractState = contract.view("state").await?.json()?;
+    let ProtocolContractState::Running(running_state) = final_state else {
+        panic!(
+            "Expected contract to be in Running state after resharing, but got: {:?}",
+            final_state
+        );
+    };
+    Ok(running_state.parameters.participants().clone())
+}
+
+pub async fn submit_tee_attestations(
+    contract: &Contract,
+    env_accounts: &mut [Account],
+    node_uids: &BTreeSet<NodeUid>,
+) -> anyhow::Result<()> {
+    env_accounts.sort_by(|a, b| a.id().cmp(b.id()));
```

Comment:
Naming aside, this can be simplified:

```suggestion
    env_accounts.sort_by_key(Account::id)
```